### PR TITLE
Get rid of manifest.json logic

### PR DESCRIPTION
**Pull request by postatum**
As discussed in https://github.com/asyncapi/tck/pull/6#issuecomment-601614506.

One downside of this is that if later we decide to add test runners for different languages, files-listing logic would have to be implemented in them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "tck",
   "description": "AsyncAPI test compatibility kit",
   "version": "1.1.0",
-  "main": "genmanifest.js tests/asyncapi-2.0",
+  "main": "",
```
Comment (fmvilas) [inline package.json]: If there's no "main", you can remove it.

File: package.json [hunk 2]
```diff
@@ -14,10 +14,5 @@
     "test framework"
   ],
   "license": "Apache-2.0",
-  "dependencies": {
-    "walk": "^2.3.14"
-  },
-  "scripts": {
-    "gen-manifest": "node genmanifest.js tests/asyncapi-2.0"
-  }
+  "dependencies": {}
```
Comment (fmvilas) [inline package.json]: Same for dependencies.

Comment (postatum) [reply package.json]: Removed.

File: package.json (hunk 1 above)
Comment (postatum) [reply package.json]: Removed.

**Comment by fmvilas** [comment]
Great stuff, Artem 👍  Feel free to merge.

**Pull request merged**